### PR TITLE
Changed the no_nii config to be create_nii to make it clearer

### PR DESCRIPTION
**Pull request by cmadjar**
In this pull request, the config setting called "no_nii" has been renamed to "create_nii" as the double negation made it hard to understand what to do.

See also:
- Redmine ticket: https://redmine.cbrain.mcgill.ca/issues/13932
- LORIS PR: https://github.com/aces/Loris/pull/3573

### CODE REVIEW

File: uploadNeuroDB/minc_insertion.pl [hunk 1]
```diff
@@ -55,7 +55,7 @@
 my $create_minc_pics    = 0;   # Default is 0, set the option to overide.
 my $globArchiveLocation = 0;   # whether to use strict ArchiveLocation strings
                                # or to glob them (like '%Loc')
-my $no_nii      = 1;           # skip NIfTI creation by default
+my $create_nii  = 1;           # skip NIfTI creation by default
```
Comment (driusan) [inline uploadNeuroDB/minc_insertion.pl]: is the fact that this default value changed intentional?

Comment (cmadjar) [reply uploadNeuroDB/minc_insertion.pl]: oops. Good catch. I changed it once and then must have cancelled the change. Changing it to be 0 as it should be.

Comment (cmadjar) [reply uploadNeuroDB/minc_insertion.pl]: @driusan Done!

**Comment by cmadjar** [comment]
@nicolasbrossard @MounaSafiHarab I rebased the branch to minor as it had conflicts. Feel free to review it again if you wish (the rebase was pretty straightforward)

**Comment by MounaSafiHarab** [comment]
tested it with and without the option of `nii` set and it works as expected.

**Pull request merged**